test/100-basic-name-resolution.bats

```diff
@@ -17,6 +17,19 @@ load helpers
 	assert "$ip_a1"
 }
 
+@test "basic container - dns itself with long network name" {
+	subnet_a=$(random_subnet 5)
+	long_name="podman1111111"
```

Comment:
Well this is no longer a name, as I said before the network name should be long but interface name must be less than 16 chars.

Reply:
Made it longer. But netavark blows up early on my local.